src/app/cli/src/client.ml

```diff
@@ -440,14 +440,17 @@ let user_command (body_args : User_command_payload.Body.t Command.Param.t)
            User_command.Payload.create ~fee ~nonce
              ~memo:User_command_memo.dummy ~body
          in
-         let payment = User_command.sign sender_kp payload in
-         dispatch_with_message Daemon_rpcs.Send_user_command.rpc
-           (payment :> User_command.t)
-           port
+         let payment =
+           (User_command.sign sender_kp payload :> User_command.t)
+         in
+         dispatch_with_message Daemon_rpcs.Send_user_command.rpc payment port
            ~success:
              (Or_error.map ~f:(fun receipt_chain_hash ->
-                  sprintf "Initiated %s\nReceipt chain hash: %s" label
-                    (Receipt.Chain_hash.to_string receipt_chain_hash) ))
+                  sprintf
+                    "Dispatched %s with ID %s\nReceipt chain hash is now %s\n"
+                    label
+                    (Receipt.Chain_hash.to_string receipt_chain_hash)
```

Comment:
You have the receipt chain hash and the base58check payment swapped here. They should switch places.